iOS Example/iOS Example.xcodeproj/project.pbxproj

```diff
@@ -85,6 +85,7 @@
 				93A3AEFC1BF142A400CAD43B /* Products */,
 			);
 			sourceTree = "<group>";
+			usesTabs = 1;
```

Comment:
No, this is intentional. The project aparently uses tab indentation but was not configured as such. This prevents commiting wrong indentation (as seen in the diff).